src/util/eventValidation.js

```diff
@@ -198,10 +198,10 @@ async function validate(event) {
           rudderValidationError = {
             type: violationTypes.RequiredMissing,
             message: error.message,
```

Comment:
Should there be parity between how the message is created for the missing vs additional property ?

Reply:
Additional Property message treats the additional message as `${error.message} : ${error.params.additionalProperty}`, should we treat the missing property message as such ?

Reply:
missing property message contains property in it by default.